libs/sample_models.py

```diff
@@ -9,13 +9,13 @@ class SampleModels:
     def __init__(self):
         logging.debug('Initializing Samples panel')
         self.items = []
-        self.items.append(neuron_utils.add_button('Very simple cell', self.loadModule, extraData = {'module': 'verysimple_cell', 'model':'VerySimpleCell'}))    
-        self.items.append(neuron_utils.add_button('Simple cell', self.loadModule, extraData = {'module': 'simple_cell', 'model':'SimpleCell'}))
+        self.items.append(neuron_utils.add_button('Single compartment Hodgkin-Huxley', self.loadModule, extraData = {'module': 'verysimple_cell', 'model':'VerySimpleCell'}))
+        self.items.append(neuron_utils.add_button('Two compartments Hodgkin-Huxley', self.loadModule, extraData = {'module': 'simple_cell', 'model':'SimpleCell'}))
```

Comment:
@tarelli If I understand the concept of compartments properly, this is not a two compartments model. We have a soma with one compartment and a dendrite with ten compartments. Btw, I didn't choose the name, I took these example from a Neuron tutorial and I kept the same names.